packages/generators/src/handlers/default.ts

```diff
@@ -16,15 +16,17 @@ export async function questions(
   self: CustomGenerator,
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
   Question: Record<string, any>,
+  forceSelection = false,
+  isSkip: Record<string, boolean> = {},
```

Comment:
Can you describe why we change it?

Reply:
So that we can reuse `default` template handler in other we based templates as base.

Reply:
oh, I see, maybe we can union two variables in one?